src/test/pagination.ts

```diff
@@ -0,0 +1,208 @@
+import axios from 'axios';
+import { sign } from 'jsonwebtoken';
+import { dataSource } from '../data-source';
+import { User } from '../user';
+import { UserInput } from '../resolvers';
+import { createFakeUser } from '../fake-user-generator';
+import { expect } from 'chai';
+
+const authenticatedUser: UserInput = {
+  name: 'Authenticated User',
+  email: 'taq@gmail.com',
+  password: 'senhafort3',
+  birthDate: '01-01-2024',
+};
+
+describe('Testing users list query with pagination', () => {
+  let token: string;
+  let users: UserInput[];
+  const dbSize = 5;
+
+  before(async () => {
+    users = [
+      await createFakeUser({ name: 'B Usuario', email: 'user0@taq.com' }),
+      await createFakeUser({ name: 'De Code', email: 'user1@taq.com' }),
+      await createFakeUser({ name: 'Ze Dev', email: 'user2@taq.com' }),
+      await createFakeUser({ name: 'Da Taq', email: 'user3@taq.com' }),
+      await createFakeUser({ name: 'A Sobrenome', email: 'user4@taq.com' }),
+    ];
```

Comment:
```suggestion
    users = await Promise.all([
      createFakeUser({ name: 'B Usuario', email: 'user0@taq.com' }),
      createFakeUser({ name: 'De Code', email: 'user1@taq.com' }),
      createFakeUser({ name: 'Ze Dev', email: 'user2@taq.com' }),
      createFakeUser({ name: 'Da Taq', email: 'user3@taq.com' }),
      createFakeUser({ name: 'A Sobrenome', email: 'user4@taq.com' }),
    ]);
```
o promise.all não tinha funcionado?